frontend/src/component/releases/ReleasePlanTemplate/TemplateForm/MilestoneList/MilestoneCard/MilestoneCard.tsx

```diff
@@ -71,9 +71,14 @@ const StyledAccordion = styled(Accordion)(({ theme }) => ({
 const StyledAccordionSummary = styled(AccordionSummary)(({ theme }) => ({
     boxShadow: 'none',
     padding: theme.spacing(1.5, 2),
+    borderRadius: theme.shape.borderRadiusMedium,
     [theme.breakpoints.down(400)]: {
         padding: theme.spacing(1, 2),
     },
+    '&.Mui-focusVisible': {
+        backgroundColor: theme.palette.background.paper,
+        padding: theme.spacing(0.5, 2, 0.3, 2),
```

Comment:
Yes, 0.3 looks weird and doesn't resolve to # of pixels properly. But for some reason this stops the size rendering from jumping around. I'll look for a better solution